assets (1).py

```diff
@@ -0,0 +1,115 @@
+from datetime import UTC, datetime, timedelta
+from typing import Any
+from fastapi import FastAPI, HTTPException
+from sqlalchemy import DateTime, ForeignKey, create_engine, select
+from sqlalchemy.ext.asyncio import async_sessionmaker, create_async_engine
+from sqlalchemy.orm import DeclarativeBase, Mapped, Session, mapped_column, relationship, sessionmaker
+
+
+from mailing import AssetMailer
+
+# Assets belongs to parent and author
+# at any given time, one parent can only have one active asset
+
+
+class Asset(DeclarativeBase):
+    __tablename__ = "asset"
+
+    id: Mapped[int]
```

Comment:
```suggestion
    id: Mapped[int] = mapped_column(Integer, primary_key=True)
```

Ca serait cool de le declarer comme primary_key, pareil pour les autres tables !